### ISSUE TITLE
Can FCP do neb calculations?

### ISSUE THREAD

**Issue by potti-charles** [hellozhaoming/FCP-vasp-ase#7]
Dear Zhaoming,

Thanks for your kind answer in my last issue, and many thanks to this useful code.

I have one question. I need do a neb calculation for activation energy in hydrogenation of CO at constant potential. Can FCP code do this? I found that the eneb code by Duan Zhiyao could do barrier at constant potential. But I have some problem in that code. I am eager to know is FCP suitable for saddle search? Thanks!

**Comment by hellozhaoming** [comment]
yes. You can use the NEB function in ASE (https://wiki.fysik.dtu.dk/ase/ase/neb.html#ase.mep.neb.NEB) to conduct a NEB calculation. You need replace the calculator by FCP, and use accurate scf coverge parameters. Since NEB is very sensitive to the forces, an accurate FCP converge should be guaranteed. If the NEB is too hard to coverge, I suggest to scan the potential surface directly by constrain a CV , for example, the distance between two atoms, and relax structures under FCP. Then you will get the relaxed FCP energy in a point of CV. change the CV, you will get the energies curve under different CVs. The energy maximum is the energy of transition state
